docs/docs/adding-a-list-of-markdown-blog-posts.md

```diff
@@ -74,4 +74,23 @@ export const pageQuery = graphql`
 `;
 ```
 
-This should get you a page with your posts sorted by descending date. You can further customise the `frontmatter` and the page component to get desired effects!
+### Creating the `PostLink` component
+
+The only thing left to do is to add the `PostLink` component. Create a new file `post-link.js` in `src/components/` and add the following:
+
+```jsx
+import React from "react";
+import Link from "gatsby-link";
+
+const PostLink = ({ post }) => (
+  <div>
+    <Link to={post.frontmatter.path}>
+      {post.frontmatter.title} ({post.frontmatter.date})
+    </Link>
+  </div>
+);
+
+export default PostLink;
+```
+
+This should get you a page with your posts sorted by descending date. You can further customise the `frontmatter` and the page and `PostLink` components to get your desired effects!
```

Comment:
this is technically correct. Another, more streamlined way to write this sentence is "You can further customize the `frontmatter`, page, and `PostLink` components to get your desired effects!"

Reply:
Is the page a component? If so, the existing wording is better.